src/GenericDecMats.jl

```diff
@@ -216,4 +90,21 @@ function generic_decomposition_matrices_overview()
     end
 end
 
+function __init__()
+    if isdefined(Main, :Oscar)
```

Comment:
```suggestion
```
Is this necessary? When I use Requires I don't have to do this.

Reply:
The point is that I do not want to include both variants, because then it is not clear which polynomials will be used.
For example, if Gapjm.jl is loaded in advance then the corresponding functionality from GenericDecMats.jl gets loaded. But if `Requires.@require` has been called also for Oscar.jl, and Oscar gets loaded later on, then also the Oscar related file would get included.

Reply:
OK, I see. Thanks for the explanation.